src/components/Swiper/index.tsx

```diff
@@ -0,0 +1,258 @@
+  transparentPagination: false,
+  threshold: 10,
+  speed: 400,
+  hasNavigation: true,
+  hasPagination: false,
+  lgSlidesSideOffset: 0,
+  smSlidesSideOffset: 0,
+} as Partial<SwiperProps>;
+
+const paginationPositionStyle: { [key in SwiperNavigationPosition]: FlattenSimpleInterpolation } = {
+  [SwiperNavigationPosition.TopRightOut]: css`
+    bottom: 16px;
+  `,
+  [SwiperNavigationPosition.BottomRightOut]: css`
+    bottom: 8px;
+  `,
+  [SwiperNavigationPosition.BottomRightIn]: css`
+    bottom: 16px;
+  `,
+};
+
+type SwiperContainerProps = Pick<
+  SwiperProps,
+  | 'hasNavigation'
+  | 'hasPagination'
+  | 'navigationPosition'
+  | 'transparentPagination'
+  | 'containerContentMaxWidth'
+  | 'paginationTheme'
+  | 'lgSlidesSideOffset'
+  | 'smSlidesSideOffset'
+>;
+
+const SwiperContainer = styled.div<SwiperContainerProps>`
+  ${props =>
+    props.lgSlidesSideOffset
+      ? css`
+          padding-left: ${props.lgSlidesSideOffset}px;
+          padding-right: ${props.lgSlidesSideOffset}px;
+        `
+      : ''};
+  ${props =>
+    props.smSlidesSideOffset
+      ? css`
+          ${media.sm`
+          padding-left: ${props.smSlidesSideOffset}px;
+          padding-right: ${props.smSlidesSideOffset}px;
+          `};
+        `
+      : ''};
+
+  .swiper-default-navigation {
+    position: absolute;
+    ${props =>
+      props.navigationPosition === SwiperNavigationPosition.TopRightOut &&
+      css`
+        top: 0;
+        transform: translateX(calc(-50% - 1px));
+      `};
+    ${props =>
+      props.navigationPosition === SwiperNavigationPosition.BottomRightOut &&
+      css`
+        top: auto;
+        bottom: 33px;
+        transform: translateX(calc(-50% - 1px));
+      `};
+    ${props =>
+      props.navigationPosition === SwiperNavigationPosition.BottomRightIn &&
+      css`
+        top: auto;
+        bottom: 47px;
+        transform: translateX(calc(-50% - 31px));
+      `};
+    left: 50%;
+    z-index: 1;
+    width: ${props => (props.containerContentMaxWidth ? `calc(100% - ${32 * 2}px)` : '100%')};
+    ${props => props.containerContentMaxWidth && `max-width: ${props.containerContentMaxWidth + 32 * 2}px`};
+    ${media.sm`
+      display: none;
+    `};
+  }
+
+  ${props =>
+    !props.hasPagination &&
+    css`
+      .swiper-pagination {
+        display: none;
+      }
+    `};
+
+  ${props =>
+    !props.hasNavigation &&
+    css`
+      .swiper-default-naviation,
+      .swiper-button-next,
+      .swiper-button-prev {
+        display: none;
+      }
+    `};
+
+  ${props =>
+    props.hasNavigation &&
+    props.navigationPosition === SwiperNavigationPosition.BottomRightOut &&
+    css`
+      padding-bottom: 48px;
+    `};
+
+  ${props =>
+    props.hasNavigation &&
+    props.navigationPosition === SwiperNavigationPosition.TopRightOut &&
+    css`
+      padding-top: 48px;
+      margin-top: -48px;
+      padding-bottom: 48px;
+    `};
+
+  ${props =>
+    props.transparentPagination &&
```

Comment:
```suggestion
    props.transparentPagination ? ~~ : ''
```

template literal 문법에서 boolean 값을 쓰면 `true` `false`그대로 문자열로 변환됩니다.
그러니 빈 문자열로 해줍시다